armi/settings/fwSettings/globalSettings.py

```diff
@@ -134,7 +134,6 @@ def defineSettings() -> List[setting.Setting]:
             CONF_INITIALIZE_BURN_CHAIN,
             default=True,
             label="Initialize Burn Chain",
-            schema=vol.Coerce(type(bool)),
```

Comment:
This was implemented in #998 but this schema was an error. The schema is auto-detected as a boolean so this is not needed. This was setting the `initializeBurnChain` setting to `bool`.